src/ui/main.qml

```diff
@@ -371,26 +355,34 @@ Window {
 
         function handleShowTour() {
             if(VPN.state === VPN.StateMain
-                && !VPNSettings.featuresTourShown
-                && newFeaturesModel.rowCount() > 0
+                && VPNWhatsNewModel.hasUnseenFeature
+                && !tourShown
             ) {
                 featureTourPopup.openTour();
             }
         }
     }
 
-    Connections {
-        target: VPNSettings
-        function onGleanEnabledChanged() {
-            Glean.setUploadEnabled(VPNSettings.gleanEnabled);
-        }
-
-        function onFeaturesTourShownChanged() {
-            featureTourPopup.handleShowTour();
+    Column {
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: 16
+        anchors.horizontalCenter: parent.horizontalCenter
+        spacing: 8
+
+        Button {
+            text: "Reset What’s new"
+            onClicked: {
+                VPNSettings.featuresTourShown = false;
+                VPNWhatsNewModel.markFeaturesAsUnseen();
+                featureTourPopup.handleShowTour();
+            }
         }
 
-        function onSeenFeaturesChanged() {
-            unseenFeaturesModel.invalidate();
+        Button {
+            text: "Mark features as seen"
+            onClicked: {
+                VPNWhatsNewModel.markFeaturesAsSeen();
+            }
         }
```

Comment:
These buttons are just here as helpers for debugging/reviewing and should be removed.